crates/core/src/factories/core_publish.rs

```diff
@@ -235,6 +321,69 @@ impl CorePublish {
             };
         }
     }
+
+    async fn outgoing_publish_agent_task(
+        mut outgoing_publish_agent_rx: Receiver<OutgoingAgentInfo>,
+        space_id: SpaceId,
+        transport: DynTransport,
+    ) {
+        while let Some((agent_info, peer_url)) =
+            outgoing_publish_agent_rx.recv().await
+        {
+            // Send fetch request to peer.
+            match serialize_publish_agent_message(&agent_info) {
+                Ok(data) => {
+                    if let Err(err) = transport
+                        .send_module(
+                            peer_url.clone(),
+                            space_id.clone(),
+                            PUBLISH_MOD_NAME.to_string(),
+                            data,
+                        )
+                        .await
+                    {
+                        tracing::warn!(
+                            ?agent_info,
+                            ?peer_url,
+                            "could not send publish agent: {err}"
+                        );
```

Comment:
```suggestion
                        tracing::debug!(
                            ?agent_info,
                            ?peer_url,
                            "could not send publish agent: {err}"
                        );
```

This is going to happen regularly when people close their laptop lids. We don't want them sending bug reports because they see warnings in the logs.